vendor.conf

```diff
@@ -49,3 +49,4 @@ github.com/klauspost/cpuid v1.2.0
 github.com/vbauerster/mpb v3.3.4
 github.com/mattn/go-isatty v0.0.4
 github.com/VividCortex/ewma v1.1.1
+github.com/DataDog/zstd 1.x
```

Comment:
`s/1.x/1.4.0/` to have an explicit version in order to avoid "accidental" updates.